.env.example

```
@@ -17,3 +17,8 @@ GEMINI_API_KEY=your_gemini_api_key_here
 # ── Other integrations (add your own below) ──────────────────────────────────
 # ANTHROPIC_API_KEY=your_anthropic_key_here   # For Claude-based workflows
 # OPENAI_API_KEY=your_openai_key_here          # Future OpenAI integration
+
+# ── Scan plugin provider Apify (Issue #325) ───────────────────────────────────────────────────────
+# Required for the Apify provider (providers/apify.mjs).
+# Get a token: https://console.apify.com/account/integrations
+# APIFY_TOKEN=your_apify_token_here
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)